services/chat/api.go

```diff
@@ -88,7 +86,7 @@ type ChannelGroup struct {
 	Chats                   map[string]*Chat                         `json:"chats"`
 	Categories              map[string]communities.CommunityCategory `json:"categories"`
 	EnsName                 string                                   `json:"ensName"`
-	Admin                   bool                                     `json:"admin"`
+	MemberRole              protobuf.CommunityMember_Roles           `json:"memberRole"`
```

Comment:
Need to make sure mobile team is aware of this change as this is breaking. /cc @cammellos

Reply:
Just realised I've already made such a comment here https://github.com/status-im/status-go/pull/3569#discussion_r1214544056

Sorry for the double ping